cli/tests/pcluster/config/test_update_policy.py

```diff
@@ -453,3 +453,357 @@ def test_compute_fleet_stop_fail_reason_and_actions_needed(
+            None,
+            False,
+            "All compute nodes must be stopped or QueueUpdateStrategy must be set",
+            "Stop the compute fleet with the pcluster update-compute-fleet command, or set QueueUpdateStrategy in the "
+            "configuration used for the 'update-cluster' operation",
+            "slurm",
+            id="running fleet and EBS changed with no update strategy set",
+        ),
+        pytest.param(
+            False,
+            "SharedStorage",
+            [],
+            None,
+            {"MountDir": "/ebs3", "Name": "ebs3", "StorageType": "Ebs", "EbsSettings": {"VolumeType": "gp3"}},
+            None,
+            None,
+            "Update actions are not currently supported for the 'SharedStorage' parameter",
+            "Restore the parameter 'SharedStorage'. If you need this change, please consider creating a new cluster "
+            "instead of updating the existing one.",
+            "awsbatch",
+            id="running fleet and EBS changed with no update strategy set with batch scheduler",
+        ),
+        pytest.param(
+            False,
+            "SharedStorage",
+            [],
+            None,
+            {"MountDir": "/ebs3", "Name": "ebs3", "StorageType": "Ebs", "EbsSettings": {"VolumeType": "gp3"}},
+            None,
+            False,
+            "All compute nodes must be stopped",
+            "Stop the compute fleet with the pcluster update-compute-fleet command",
+            "plugin",
+            id="running fleet and EBS changed with no update strategy set with plugin scheduler",
+        ),
+        pytest.param(
+            False,
+            "SharedStorage",
+            [],
+            None,
+            {"MountDir": "/ebs3", "Name": "ebs3", "StorageType": "Ebs", "EbsSettings": {"VolumeType": "gp3"}},
+            QueueUpdateStrategy.TERMINATE.value,
+            True,
+            None,
+            None,
+            "slurm",
+            id="running fleet and Ebs added with update strategy TERMINATE",
+        ),
+        pytest.param(
+            False,
+            "SharedStorage",
+            [],
+            {"MountDir": "/ebs3", "Name": "ebs3", "StorageType": "Ebs", "EbsSettings": {"VolumeType": "gp3"}},
+            None,
+            QueueUpdateStrategy.DRAIN.value,
+            False,
+            "All compute nodes must be stopped",
+            "Stop the compute fleet with the pcluster update-compute-fleet command",
+            "slurm",
+            id="running fleet and Ebs removed with update strategy DRAIN",
+        ),
+        pytest.param(
+            False,
+            "SharedStorage",
+            [],
+            None,
+            {"MountDir": "/efs4", "Name": "efs4", "StorageType": "Efs"},
+            QueueUpdateStrategy.TERMINATE.value,
+            True,
+            None,
+            None,
+            "slurm",
+            id="running fleet and EFs added with update strategy DRAIN",
```

Comment:
The update strategy is actually TERMINATE, not DRAIN.

Reply:
Changed the id